utils/src/health.rs

```diff
@@ -0,0 +1,60 @@
+use crate::consts::{DEFAULT_BACKUP_DIR, DEFAULT_DATA_DIR};
+use std::fs::{read_to_string, remove_file, write};
+use std::path::Path;
+
+/// The text to test the volume is working fine, just for testing purposes :)
+const TEST_TEXT: &str = "If I become a cat, I won't have to work anymore :)";
+/// The test file name
+const TEST_FILENAME: &str = "working_mans_dream";
+
+/// Check if the volume under the path is working fine
+fn check_volume(path: &Path) -> bool {
```

Comment:
K8s seems to provide a volume health monitoring feature. FYI: https://kubernetes.io/docs/concepts/storage/volume-health-monitoring/

Reply:
The k8s volume health monitoring can be implemented in various ways as mentioned in the documentation. But not all of these methods are enabled by default. Simply reading and writing a file here is a very convenient approach to check volume health.